src/Scripting/VM/VMException.hpp

```
@@ -0,0 +1,76 @@
+	}
+
+private:
+	std::string m_excMsg;
+};
+
+
+/* Virtual Machine Breakpoint Trap */
+class VMBreakpointTrap : public std::exception {
+public:
+	VMBreakpointTrap(size_t progCounter) : m_pc(progCounter) {}
+	~VMBreakpointTrap() = default;
```

Clang (14.0.6)

[warning] annotate this function with 'override' or (rarely) 'final'

(cppcoreguidelines-explicit-virtual-functions,modernize-use-override)